Tracks/kaddaok_Import UVR separations into comping lane structure.lua

```diff
@@ -1,17 +1,27 @@
--- @description Import UVR separations into comping lane structure
+-- @description Import UVR or X-Minus separations into comping lane structure
```

Comment:
Is updating this `@description` an issue?  What will that mean for anyone currently using the script; does it still associate as being out of date?

Reply:
No, it's fine. `@description` is only used for display and searching. As long as the file is not renamed, there is no side-effects.